scriptmodules/supplementary/skyscraper.sh

```diff
@@ -469,89 +419,85 @@ function gui_skyscraper() {
+        [1]="Gather and cache resources for all systems found in \Zb$romdir\Zn."
+        [2]="Choose which systems will have resources gathered and cached, from the ones found in \Zb$romdir\Zn."
+        [3]="Select the source for ROM scraping. Supported sources:\n\ZbONLINE\Zn\n * ScreenScraper (screenscraper.fr)\n * TheGamesDB (thegamesdb.net)\n * OpenRetro (openretro.org)\n * ArcadeDB (adb.arcadeitalia.net)\n * World of Spectrum (worldofspectrum.org)\n\ZbLOCAL\Zn\n * Emulationstation Gamelist (imports data from ES gamelist)\n * Import (imports resources in the local cache)\n\n\Zb\ZrNOTE\Zn: Some sources require a username and password for access. These can be set per source in the \Zbconfig.ini\Zn configuration file.\n\n Skyscraper parameter: \Zb-s <source_name>\Zn"
+        [4]="Options for resource gathering and caching sub-menu.\nClick to open it."
+        [5]="Generate Emulationstation game lists.\nRuns the scraper to incorporate downloaded information and media from the local cache and write them to \Zbgamelist.xml\Zn files to be used by Emulationstation."
+        [6]="Options for Emulationstation game list generation sub-menu.\nClick to open it and change the options."
+        [V]="Toggle the download and caching of videos.\nThis also toggles whether the videos will be included in the resulting gamelist.\n\nSkyscraper option: \Zb--videos\Zn"
         [A]="Advanced options sub-menu."
+        [U]="Check for an update to Skyscraper\nIf there is a new release, you'll have the option to update."
     )
 
     ver=$(_get_ver_skyscraper)
 
     while true; do
         [[ -z "$ver" ]] && ver="v(Git)"
 
-        local cmd=(dialog --backtitle "$__backtitle"  --colors --cancel-label "Exit" --help-button --no-collapse --cr-wrap --default-item "$default" --menu "   Skyscraper: game scraper by Lars Muldjord ($ver)\\n \\n" 21 65 10)
+        local cmd=(dialog --backtitle "$__backtitle"  --colors --cancel-label "Exit" --help-button --no-collapse --cr-wrap --default-item "$default" --menu "   Skyscraper: game scraper by Lars Muldjord ($ver)\\n \\n" 22 60 13)
 
         local -a options=(
-            1 "Scrape all systems"
-            2 "Scrape chosen systems"
+            "-" "GATHER and cache resources"
         )
 
         local source_found=0
         local online="Online"
         local i
 
+        options+=(
+            1 "Gather for All systems"
+            2 "Gather for Chosen systems"
+        )
+
         for i in "${!s_source[@]}"; do
             if [[ "$scrape_source" == "${s_source[$i]}" ]]; then
                 [[ $i -ge 10 ]] && online="Local"
-                options+=(3 "Scrape from ${s_source_names[$i]} ($online)")
+                options+=(3 "Gather source - ${s_source_names[$i]} ($online) -->")
                 source_found=1
             fi
         done
 
-        if [[ ! $source_found ]]; then
-            options+=(3 "Scrape from Screenscraper (Default)")
+        if [[ $source_found -ne 1 ]]; then
```

Comment:
not sure if `-ne` or `!=` is preferred

Reply:
after checking the RetroPie source code, looks like `-ne` is preferred for numbers indeed, so please ignore this comment :)